external/java/src/main/java/ch/epfl/dedis/lib/omniledger/OmniledgerRPC.java

```diff
@@ -82,12 +83,16 @@ public OmniledgerRPC(Roster r, Darc d, Duration blockInterval) throws CothorityE
      */
     public OmniledgerRPC(Roster roster, SkipblockId skipchainId) throws CothorityException, InvalidProtocolBufferException {
         Proof proof = OmniledgerRPC.getProof(roster, skipchainId, InstanceId.zero());
-        OmniledgerRPC.checkProof(proof, "config");
-        config = new Config(proof.getValues().get(0));
+        if (!proof.isContract("config", skipchainId)){
```

Comment:
General remark - it will be useful to extract contracts names like "config", "darc" ... as constants or as some kind of class.